src/sentry/static/sentry/app/views/settings/project/projectFilters/projectFiltersChart.jsx

```diff
@@ -122,17 +128,17 @@ const ProjectFiltersChart = createReactClass({
           this.setState({error: true});
         }.bind(this)
       );
-  },
+  }
 
   fetchData() {
     this.getFilterStats();
-  },
+  }
```

Comment:
`<LoadingError>` calls `fetchData`, does `this` need to be bound?

Reply:
Thanks, it does need to be bound.